pkgs/tools/system/krakenx/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, python3, python3Packages }:
```

Comment:
You can remove `python3` since it is not used

Reply:
Done. I was using `python3.pkgs` at first and forgot to remove it there %)